src/infra/modules/frontend/main.tf

```diff
@@ -278,6 +303,45 @@ module "webacl_count_alarm" {
   alarm_actions = [var.web_acl.sns_topic_arn]
 }
 
+module "rest_api_admin" {
+  source = "../rest-api"
+
+  name                 = var.rest_api_admin_name
+  stage_name           = var.rest_api_admin_stage
+  xray_tracing_enabled = var.xray_tracing_enabled
+
+  body = templatefile(var.openapi_admin_template_file,
+    {
+      server_url                   = var.domain_name
+      uri                          = format("http://%s:%s", var.nlb_dns_name, "8080"),
```

Comment:
As far as I can see there is not integration with the internal load balancer. According to the openapi the only available integration is with lambda functions.